lib/src/dispatcher.dart

```diff
@@ -56,17 +56,6 @@ class Dispatcher {
         }
 
         switch (message.whichMessage()) {
-          case InboundMessage_Message.error:
```

Comment:
Can you also update the protocol buffer definition to drop `InboundMessage.message.error`?

Reply:
Ah, good point.